bin/countly.install_ubuntu.sh

```diff
@@ -118,5 +118,6 @@ then
     else
         echo "Cannot reach Google. Disabling Google services. You can enable this from Configurations later."
         countly config "frontend.use_google" false
+        countly plugin enable EChartMap
```

Comment:
It would be great if we could do that before generating production files (`cd $DIR && grunt dist-al`), or else we need to generate them here again